specification/azsadmin/resource-manager/user-subscriptions/Microsoft.Subscriptions/preview/2015-11-01/Subscriptions.json

```diff
@@ -0,0 +1,412 @@
+                            },
+                            "provider": {
+                                "description": "Resource provider display name.",
+                                "type": "string",
+                                "readOnly": true
+                            },
+                            "resource": {
+                                "description": "Resource type display name related to the operation.",
+                                "type": "string",
+                                "readOnly": true
+                            }
+                        }
+                    },
+                    "name": {
+                        "description": "Name of the operation being performed on an object.",
+                        "type": "string",
+                        "readOnly": true
+                    },
+                    "origin": {
+                        "description": "Origin for the operation.",
+                        "type": "string",
+                        "readOnly": true
+                    }
+                }
+            },
+            "OperationList": {
+                "description": "List of supported operations.",
+                "type": "object",
+                "properties": {
+                    "value": {
+                        "description": "List of supported operations.",
+                        "type": "array",
+                        "items": {
+                            "$ref": "#/definitions/Operation"
+                        }
+                    }
+                }
+            }            
+        },
+        "parameters": {
+            "SubscriptionIdParameter": {
+                "name": "subscriptionId",
+                "in": "path",
+                "description": "Subscription credentials which uniquely identify Microsoft Azure subscription.The subscription ID forms part of the URI for every service call.",
+                "required": true,
+                "type": "string"
+            },
+            "ApiVersionParameter": {
+                "name": "api-version",
+                "in": "query",
+                "description": "Client Api Version.",
+                "required": true,
+                "type": "string",
+                "default": "2015-11-01"
+            },
+            "ResourceGroupParameter": {
+                "description": "The resource group the resource is located under.",
+                "name": "resourceGroup",
+                "in": "path",
+                "type": "string",
+                "required": true,
+                "x-ms-parameter-location": "method"
+            },
+            "LocationParameter": {
+                "description": "The AzureStack location.",
+                "name": "location",
+                "in": "path",
+                "type": "string",
+                "required": true,
+                "x-ms-parameter-location": "method"
+            },
+            "SubscriptionParameter": {
```

Comment:
I don't see where this is being referenced.

Reply:
removed